LICENSE

```diff
@@ -202,15 +202,41 @@
    limitations under the License.
 
 
+================================================================================
+
+For the `enteprise` directory:
```

Comment:
enterprise

Reply:
the backticks here are for consistency with the other entry in this file

see here: https://github.com/crate/crate/blob/588259c7e4e26b81ef336fef39d42e596c104702/LICENSE#L232

using backticks to quote the directory name as a literal

Reply:
i meant the spelling of enterprise :)